packages/web/components/Technology/Details/Header.js

```diff
@@ -32,6 +33,8 @@ const Header = () => {
 		return openModal(name, { technology });
 	};
 
+	const ownerUser = technology.users?.find((tUser) => tUser?.pivot?.role === ROLES.OWNER);
```

Comment:
I guess that would be interesting to move this function to a helper
```suggestion
	const ownerUser = technology.users?.find((user) => user?.pivot?.role === ROLES.OWNER);
```

Reply:
"User is already declared in the upper scope :D"

Agree about moving to a helper